.gitignore

```diff
@@ -34,3 +34,4 @@
 public/sitemap.xml
 public/system/
 /public/ckeditor_assets/
+.idea/
```

Comment:
I think this no necessary add this change on the PR 😉

Reply:
Hi @decabeza 
I have done the things you have asked, I hope next fix would be with no problem ;).
Thank you